bodybank/character_generator/models.py

```diff
@@ -107,4 +107,20 @@ class Background2Skills(models.Model):
     modifier = models.PositiveIntegerField(default=0)
 
     def __str__(self):
-        return (' '.join([str(self.background), str(self.skills), str(self.modifier)]))
\ No newline at end of file
+        return (' '.join([str(self.background), str(self.skills), str(self.modifier)]))
+
+
+class Career(models.Model):
+    name = models.CharField(max_length=100)
+    desc = models.TextField("Description", default="")
+
+    def __str__(self):
+        return self.name
+
+class Career2Skills(models.Model):
+    career = models.ForeignKey(Career, related_name = 'career_to_skill', on_delete=models.CASCADE)
+    skills = models.ForeignKey(Skill, related_name='skill_to_career', on_delete=models.CASCADE)
+    modifier = models.PositiveIntegerField(default=0)
```

Comment:
is there a better name for this or is it called like that in the game?

Reply:
Its called career in the game :raccoon: